src/main/java/com/fundation/logic/common/MetadataImageExtractor.java

```diff
@@ -60,39 +60,41 @@ public void readAll() {
 
     /**
      * This method read mime type.
-     *
-     * @return
+     * @return void
      */
     public static void mimeType(String mimeType) {
```

Comment:
Use descriptive method names as it is difficult to understand what the method does just for the name